.hintrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
- [ ] Your linters are failing. Linter result should be green, this means that the last commit have a green check mark (✅) or a summary of the passed checks is in the PR.
- [ ] Please add a descriptive title and a short summary to your pull request. In order to make your repo look professional, pull request should have a descriptive title telling you what changes are inside and pull request should have a short summary describing introduced changes in more detail.